src/CattyTests/Bricks/WaitBrickTests.swift

```diff
@@ -156,4 +156,12 @@ final class WaitBrickTests: XCTestCase {
         XCTAssertFalse(waitBrick.timeToWaitInSeconds.isEqual(to: Formula(float: 1)))
     }
 
+    func testUnknownType() {
```

Comment:
This test is not really testing your new functionality. You can remove this test.